src/index.html

```diff
@@ -3,11 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="Product Image"
+          class="card__logo"
+        >
+      </div>
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br>5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__name">
+        <div class="card__product-code">Product code: 195434</div>
+      </div>
+
+      <div class="stars">
+        <div class="stars__block">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <div class="stars__review">Reviews: 5</div>
+      </div>
```

Comment:
Stas should wokr with modifiers 1-5
```suggestion
      <div class="raiting">
        <div class="stars">
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
        </div>

        <p class="raiting__review">Reviews: 5</p>
      </div>
```